neo/rawio/blackrockrawio.py

```diff
@@ -346,52 +352,94 @@ def _parse_header(self):
             # t_start/t_stop for segment are given by nsx limits or nev limits
             self._sigs_t_starts = []
             self._seg_t_starts, self._seg_t_stops = [], []
+            nb_empty_segments = 0
             for data_bl in range(self._nb_segment):
                 length = self.nsx_data[data_bl].shape[0]
+                # Discard empty segments
+                if length < 2:
+                    nb_empty_segments += 1
+                    self.nsx_data.pop(data_bl)
+                    # TODO: CHECK WHAT HAPPENS HERE AND IF -1 IS CORRECT
```

Comment:
is this comment still valid?
I think subtracting 1 of data_bl is not the right solution since it is only providing the reset of the counter in the current run of the loop, but in the next run the index is continued as usual. Maybe it would be better to use while here instead of a for loop.